payslip_line_manually/models/hr_salary_rule.py

```diff
@@ -0,0 +1,31 @@
+from odoo import fields, models
+from odoo.tools.safe_eval import safe_eval
+
+
+class HrSalaryRule(models.Model):
+    _inherit = "hr.salary.rule"
+
+    analytic_account_id = fields.Many2one(
+        "account.analytic.account", "Analytic Account"
+    )
+    line_manually_model = fields.Selection(
+        [("hr.contract", "Contract"), ("hr.payslip", "Payslip")],
+        string="Salary Rule input from",
+    )
+
+    def _compute_values_python(self, localdict):
+        localdict["rule"] = self
+        safe_eval(self.amount_python_compute, localdict, mode="exec", nocopy=True)
+        if localdict.get("result_list"):
+            return localdict["result_list"]
+        return {
+            "name": localdict.get("result_name") or False,
+            "analytic_account_id": localdict.get("result_analytic") or False,
```

Comment:
If you are only adding this to the calculation, you can use super() and only add the value to the returning dict, or am i missing something?